### ISSUE TITLE
Modificar rutas de archivos estáticos. 

### ISSUE THREAD

**Issue by abr4xas** [ngVenezuela/enki#9]
Recomendaría no usar `./src` en las rutas es mejor usar `src`.

Adicional a esto, tambien creo conveniente cambiar el nombre de `statics` a `assets` y ahi incluir todo el contenido estático del proyecto (css, imágenes, js).

@Villanuevand @jobsamuel @JMEspiz

**Comment by jobsamuel** [comment]
Considero que cambiar de `statics` a `assets` es bienvenido; apoyo la idea. Sin embargo, `css` y `js` deberían permanecer como están y sólo dedicar `assets` para el resto de contenido estático (imágenes, PDFs, TXTs, etc).

**Comment by JMEspiz** [comment]
Concuerdo con @abr4xas y @jobsamuel pasar statics a assets. Considero al igual que Jobs que los _css_ y _js_ queden dentro de src.

**Comment by jobsamuel** [comment]
Hey @JMEspiz, fue @abr4xas quien sugirió que `css` y `js` estén dentro de `assets`. **Yo no** :sunglasses:

**Comment by Villanuevand** [comment]
Entonces quedaría `assets/js/*.js` y `assets/css/*.css`. ??? 
@abr4xas @jobsamuel @JMEspiz

**Comment by abr4xas** [comment]
Si, así mismo.

**Comment by jobsamuel** [comment]
Sí, quedaría así @Villanuevand. Pero lo que comenté (y @JMEspiz apoyó) fue:

> Sin embargo, css y js deberían permanecer como están y sólo dedicar assets para el resto de contenido estático (imágenes, PDFs, TXTs, etc).

Es decir, `src/css/*.css` y `src/js/*.js`

Lo que @abr4xas propuso **yo no lo apoyo** :sunglasses:

**Comment by JMEspiz** [comment]
LOL xD.

> `Es decir, src/css/*.css y src/js/*.js`

Me parece mejor. Y que en `assets` tenga las imagenes y demás archivos.
